cibuildwheel/windows.py

```diff
@@ -62,17 +63,29 @@ def simple_shell(args, env=None, cwd=None):
         print('+ ' + ' '.join(args))
         args = ['cmd', '/E:ON', '/V:ON', '/C'] + args
         return subprocess.check_call(' '.join(args), env=env, cwd=cwd)
+    
     def download(url, dest):
         print('+ Download ' + url + ' to ' + dest)
         dest_dir = os.path.dirname(dest)
         if not os.path.exists(dest_dir):
             os.makedirs(dest_dir)
-        response = urlopen(url)
+        for _ in range(3):
+            try:
+                response = urlopen(url)
+            except:
+                sleep(3)
+                continue
+            break
+        else:
+            print("Download from url " + url + "failed", file=sys.stderr)
```

Comment:
Could we log the final exception here, too? Could be useful for debugging if the problem isn't a transient one

Reply:
I change behaviour to re raise last exception.